crates/iceberg/src/spec/manifest_list.rs

```diff
@@ -69,6 +73,25 @@ impl ManifestList {
         &self.entries
     }
 
+    /// Get the v1 schema of the manifest list entry.
+    pub(crate) fn v1_schema() -> Schema {
```

Comment:
How about narrowing down it to `private`?

Reply:
I moved the schema definitions into `_const_fields`.